src/alza.js

```diff
@@ -1,6 +1,7 @@
-import _request from 'request'
 import cheerio from 'cheerio'
+import _request from 'request'
```

Comment:
✨ NIT: `request` is no longer under development and should not be used.

ref: https://github.com/request/request/issues/3142

Use [`got`](https://github.com/sindresorhus/got) wherever possible instead.